Source/WebKit/UIProcess/API/gtk/WebKitWebView.h

```diff
@@ -293,6 +293,7 @@ struct _WebKitWebViewClass {
                                                 WebKitWebProcessTerminationReason reason);
     gboolean   (* user_message_received)       (WebKitWebView               *web_view,
                                                 WebKitUserMessage           *message);
+    void       (* initialize_web_extensions)   (WebKitWebView               *web_view);
 
     /*< private >*/
     void (*_webkit_reserved0) (void);
```

Comment:
Careful, you have to remove the last padding byte `_webkit_reserved0` here when adding your new signal, or else you break ABI and applications will crash!

Congrats on adding the final signal to WebKitWebViewClass. We'll need to add more padding for the future in https://bugs.webkit.org/show_bug.cgi?id=240472.